language-design.md

```diff
@@ -28,7 +65,28 @@ you know a poorly designed language? What are the symptoms?
 
 **Response**
 
-
+A well designed language is designed for growth. It should be clear what the
+rules are for extending a language, how to use the tools built into the language, 
+and what the code means. It should provide the tools necessary for the general
+purposes of the majority of the intended user base, and cover the intended use
+cases of the language, but not provide tools for niche use cases. However, it 
+should be easy and clear how to implement the tools needed for the niche cases.
+
+A well designed language should cover its use cases thoroughly and simply as well.
+If something is an intended use case of the language, it should be well supported
+and not require the user to stray too far from the overall design of the language.
+If a language requires the user to jump through hoops to do something that is part
+of the goal of the language, then I think it is a sign of poor design.
```

Comment:
How do you define the "goal" or "intended use case" of a language that is meant to be very general, like C or Java?